Composer/packages/electron-server/src/main.ts

```diff
@@ -295,11 +295,16 @@ async function run() {
     setTimeout(() => startApp(signalThatMainWindowIsShowing), 500);
 
     const mainWindow = getMainWindow();
+    const machineId = await getMachineId();
+
     mainWindow?.webContents.send('session-update', 'session-started');
 
     if (process.env.COMPOSER_DEV_TOOLS) {
       mainWindow?.webContents.openDevTools();
     }
+
+    log(`Machine ID is ${machineId}`);
```

Comment:
Is this log required?

Reply:
It's not required, but it's useful - this goes into the debug log which users won't see unless they're running via the console anyway.

Reply:
Logs are only visible from the terminal, so I think it's fine.